std/container/rbtree.d

```diff
@@ -1094,6 +1094,110 @@ if (is(typeof(binaryFun!less(T.init, T.init))))
         assert(t1 != o);  // pathological case, must not crash
     }
 
+    /**
+     * Generates a hash for a Red Black Tree
+     *
+     *
```

Comment:
This block comment is unnecessary big and quite pointless. 

> for a Red Black Tree

The reader knows that already, it's an overridden member function after all.

Reply:
I fixed the comment.